tests/foreman/api/test_provisioning.py

```diff
@@ -592,6 +594,112 @@ def test_rhel_pxe_provisioning_fips_enabled(
     assert provisioning_host.subscribed, 'Host is not subscribed'
 
 
+@pytest.mark.e2e
+@pytest.mark.upgrade
+@pytest.mark.on_premises_provisioning
+@pytest.mark.parametrize('pxe_loader', ['secureboot'], indirect=True)
+@pytest.mark.rhel_ver_match([get_sat_rhel_version().major])
```

Comment:
why not `@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])` ?

Reply:
Currently its covering a scenario which covers provisioning for content_host with rhel_version same as get_sat_rhel_version().major, which is only supported scenario now